common/views/components/Dot/Dot.tsx

```diff
@@ -17,8 +17,8 @@ type Props = {
   color: PaletteColor;
 };
 
-const Dot: FunctionComponent<Props> = ({ color }: Props) => {
-  return <DotEl className={classNames({ [`font-${color}`]: true })} />;
-};
+const Dot: FC<Props> = ({ color }) => (
+  <DotEl className={classNames({ [`font-${color}`]: true })} />
+);
```

Comment:
this is completely irrelevant, but I wanted to trigger a new Chromatic build

Reply:
I see both FC and FunctionComponent used and I've been trying to figure out why one and not the other, internet seems to lean towards FunctionComponent, do we have a preference?

Reply:
I feel like we've been leaning towards `FC` in our codebases. I don't think it makes a huge difference; we should pick one and standardise on it.